contracts/src/bin/vendor.rs

```diff
@@ -46,6 +46,10 @@ fn run() -> Result<()> {
             "balancer-labs/balancer-v2-monorepo/a3b570a2aa655d4c4941a67e3db6a06fbd72ef09/\
              pkg/deployments/deployed/mainnet/WeightedPool2TokensFactory.json",
         )?
+        .github(
+            "BalancerV2StablePoolFactory",
+            "balancer-labs/balancer-subgraph-v2/2b97edd5e65aed06718ce64a69111ccdabccf048/abis/StablePoolFactory.json",
+        )?
```

Comment:
AFAIK, this is `abi_only` right? THere doesn't seem to be any bytecode in the vendored file.

Reply:
I guess not... in that case I don't really know where to get it from. This has become quite a nuisance. Should I just find the bytecode and paste it in?

Reply:
We don't make use of the bytecode anywhere AFAIK, why not just include it in the `abi_only()` vendored contracts (like the `BalancerV2StablePoolFactory`)? My point is that right now its included as a `full()` vendored contract but doesn't need to be.

Reply:
Alright I found it.

https://github.com/balancer-labs/balancer-v2-monorepo/blob/stable-deployment/pkg/deployments/tasks/20210624-stable-pool/abi/StablePoolFactory.json

Reply:
But we will use it later... why not just include it now.